{"name": "Проверка получения заказа по несуществующему id", "status": "failed", "statusDetails": {"message": "AssertionError: ОР: 200\nФР: 404\nmessage: b'{\"code\":1,\"type\":\"error\",\"message\":\"Order not found\"}'", "trace": "self = <test_store_api.TestPetApi object at 0x101e47640>\n\n    @allure.title('Проверка получения заказа по несуществующему id')\n    def test_get_store_inventory(self):\n        store_api = StoreApi()\n        response = store_api.get_store_order_by_id(57668486, expected_error=True)\n>       check.status_code(200, response)\n\nresponse   = <Response [404]>\nself       = <test_store_api.TestPetApi object at 0x101e47640>\nstore_api  = <api_testing.store_api.StoreApi object at 0x101e47b50>\n\n/Users/annapogrebnik/PycharmProjects/pythonProject1/otus_test_project/tests/test_store_api.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/Users/annapogrebnik/PycharmProjects/pythonProject1/otus_test_project/helpers/checker.py:25: in status_code\n    _response_data(expected, res_code, 'status_code', res_text)\n        actual     = <Response [404]>\n        expected   = 200\n        res_code   = 404\n        res_text   = b'{\"code\":1,\"type\":\"error\",\"message\":\"Order not found\"}'\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nexpected = 200, actual = 404, step_text = 'status_code'\nmessage = b'{\"code\":1,\"type\":\"error\",\"message\":\"Order not found\"}'\n\n    def _response_data(expected, actual, step_text, message):\n        with allure.step(f\"Проверка {step_text} ответа\"):\n            if isinstance(expected, int):\n>               assert actual == expected, \\\n                    (f\"ОР: {expected}\\n\"\n                     f\"ФР: {actual}\\n\"\n                     f\"message: {message}\")\nE               AssertionError: ОР: 200\nE               ФР: 404\nE               message: b'{\"code\":1,\"type\":\"error\",\"message\":\"Order not found\"}'\n\nactual     = 404\nexpected   = 200\nmessage    = b'{\"code\":1,\"type\":\"error\",\"message\":\"Order not found\"}'\nstep_text  = 'status_code'\n\n/Users/annapogrebnik/PycharmProjects/pythonProject1/otus_test_project/helpers/checker.py:8: AssertionError"}, "steps": [{"name": "Проверка status_code ответа", "status": "failed", "statusDetails": {"message": "AssertionError: ОР: 200\nФР: 404\nmessage: b'{\"code\":1,\"type\":\"error\",\"message\":\"Order not found\"}'\n", "trace": "  File \"/Users/annapogrebnik/PycharmProjects/pythonProject1/otus_test_project/helpers/checker.py\", line 8, in _response_data\n    assert actual == expected, \\\n"}, "start": 1714319146141, "stop": 1714319146141}], "start": 1714319145467, "stop": 1714319146142, "uuid": "b4d6be93-8a29-489c-a1dc-61e04cf7b511", "historyId": "fd395fb132282755c495bd57f8fec1ef", "testCaseId": "fd395fb132282755c495bd57f8fec1ef", "fullName": "tests.test_store_api.TestPetApi#test_get_store_inventory", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_store_api"}, {"name": "subSuite", "value": "TestPetApi"}, {"name": "host", "value": "d45763e349a4"}, {"name": "thread", "value": "94517-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_store_api"}]}